### ISSUE TITLE
Toggle outputs

### ISSUE THREAD

**Issue by tremby** [ambientsound/pms#71]
It'd be great to have an interface within PMS to toggle outputs on and off.

Like `mpc outputs`, `mpc disable 2`, etc

**Comment by ambientsound** [comment]
Yes, indeed. Either as a command-line interface, a list-style view, or both. In case of a list view, some architecture changes are needed in order to abstract away the _list_ functionality.

**Comment by tremby** [comment]
I think this one is beyond me for now. Though if you were to get me started, such as getting me as far as a new blank widget which I can then add things to, I could probably do the rest.

**Comment by tremby** [comment]
Oh, and compact topbar fields for which outputs are on and off would be nice. One-character red/green for each output, perhaps.
